### PR TITLE
Log 'nowplaying' track to file

### PR DESCRIPTION
**Pull request by nickspoons**
I don't know if you're at all interested in merging this, but it's a feature I wanted (for displaying currently playing track in i3blocks) so I added it for myself.

When configured, the current track will be logged to a file, allowing the user the display the music status externally, e.g. as a status bar element or system notification.

### CODE REVIEW

**Comment by christopher-dG** [comment]
I think you may have forgotten to check in a file, should there by a `nowplaying.py` at the package root?

**Comment by nickspoons** [comment]
Oops ... yes there should. And I've shut everything down for the night, it'll have to wait until the morning, sorry about that.

**Comment by christopher-dG** [comment]
Haha no problem 🙂. It's a good feature though, I thought about adding it too.

As a side note, I'm beginning the slow process of redesigning/rebuilding this project so it might not be worth investing too much time into adding features at the moment (although I do appreciate the PRs).

**Comment by nickspoons** [comment]
Yeah I saw that you were working on an overhaul. That's fine, I'm just tweaking to support my own preferences, and merging back anything useable. But I think I'm done for now anyway!

**Comment by nickspoons** [comment]
Checked in the missing file, thanks @christopher-dG

File: README.md [hunk 1]
```diff
@@ -64,6 +64,19 @@ colour, which means this allows transparency if your terminal supports it.
 be modified. Just open a new terminal session and your colours will
 be back to normal.
 
+### Now playing
```
Comment (christopher-dG) [inline README.md]: Now **P**laying please 🙂

File: README.md [hunk 2]
```diff
@@ -64,6 +64,19 @@ colour, which means this allows transparency if your terminal supports it.
...
+
+To log the currently playing track to a file, include a `nowplaying` section in
+your config file with `enable` set to `yes`. A `filename` may then be specified,
+or the default value of `~/.nowplaying` will be used.
+
+This file may then be used in status bars or simple notifications systems. For
+example, here is a simple i3blocks definition:
+
+    [music]
```
Comment (christopher-dG) [inline README.md]: Can you put this in triple backticks instead of indented? i.e.:

\```
stuff
\```

**Review by christopher-dG — CHANGES_REQUESTED** [review]
Looks great! Just a couple of nitpicks.

**Comment by nickspoons** [comment]
I fixed the README formatting. Are you happy with config values `nowplaying` and `filename` or would you prefer camelCase or snake_case or something? What about the default file `~/.nowplaying`?

**Review by christopher-dG — APPROVED** [review]

**Comment by christopher-dG** [comment]
The naming is fine with me. Thanks for your contributions!

**Pull request merged**